packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -318,6 +318,16 @@ const logRequest = (request, {header = '', success = '', failure = ''}) => {
  * @event meeting:self:lobbyWaiting
  * @instance
  * @type {Object}
+ * @property {Object} reason Reason why user left the meeting
+ * @memberof Meeting
+ */
+
+/**
+ * Meeting Self Left State
+ * Emitted when user leaves the meeting either by own/server kicking off the user/user not active for 40 seconds or more
```

Comment:
the comment says that this event is emitted when "user leaves the meeting either by own...." - does this mean that it should be emitted when client calls `leave()`? Currently the code doesn't do that.

Reply:
yes you are correct , I still need to add a JIRA when a user gets kicked off by the server manually